src/main/java/org/eclipse/foundation/marketplace/config/JsonBConfig.java

```diff
@@ -4,7 +4,7 @@
  * which is available at http://www.eclipse.org/legal/epl-v20.html,
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.eclipsefoundation.marketplace.config;
+package org.eclipse.foundation.marketplace.config;
```

Comment:
actually, I liked better the eclipsefoundation (single word) package name. Why did you change that?

Reply:
I matched the package naming found within the ECA plugin, but I can see now that the bot api that you created has single word...we should choose which we are moving forward with for future APIs. I agree that one word looks nicer. Should I revert that change?

Reply:
I did not create the ECA plugin, so I've no history about the choosing of package name. I think that we should stick to "eclipsefoundation" for all of our future work. If it's not too much burden, yes please revert the package rename. Thanks.